esx_service/volume_kv.py

```diff
@@ -39,6 +39,8 @@
 ATTACHED_VM_UUID = 'attachedVMUuid'
 # The name of the VM that the volume is attached to
 ATTACHED_VM_NAME = "attachedVMName"
+# The device to which the volume is attached.
```

Comment:
Hi,

I understand the comment is not in the same style as those above it, but those above do not adhere proper grammar as they end in prepositions. The proper form is _to which the X is Y_, not _the X is Y to_.